Provider/DataTypeProvider.php

```diff
@@ -80,6 +80,14 @@ public function getDataTypes()
                 'entity_types' => ['application'],
                 'order' => 5
             ],
+            [
```

Comment:
Is default value not possible here?

Reply:
Not as far as I can tell. @PieterDigipolis ?